### PR TITLE
Add streak model

### PR DESCRIPTION
**Pull request by ignaoya**

### CODE REVIEW

File: tracker/migrations/0002_streak.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    dependencies = [
+        ('tracker', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Streak',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('start_date', models.DateTimeField(default=datetime.datetime(2022, 11, 24, 15, 17, 56, 608448, tzinfo=datetime.timezone.utc))),
```
Comment (nicky-eng) [inline tracker/migrations/0002_streak.py:19]: datetime causing migration warnings. Please, resolve!

File: tracker/migrations/0002_streak.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+                ('end_date', models.DateTimeField(default=datetime.datetime(2022, 11, 24, 15, 17, 56, 608505, tzinfo=datetime.timezone.utc))),
```
Comment (nicky-eng) [inline tracker/migrations/0002_streak.py:20]: Same here!

File: tracker/models.py [hunk 3]
```diff
@@ -27,6 +29,15 @@ def __str__(self):
         return f"{self.name} - {self.user}"
 
 
+class Streak(models.Model):
+    habit = models.ForeignKey(Habit, on_delete=models.CASCADE, related_name='streaks')
+    start_date = models.DateTimeField(default=timezone.now())
```
Comment (nicky-eng) [inline tracker/models.py:34]: Fix this to fix migrations

**Review by nicky-eng — CHANGES_REQUESTED** [review]

**Review by nicky-eng — APPROVED** [review]
Can you post the StackOverflow link you followed to solve this problem?

**Comment by ignaoya** [comment]
The resolution was found in the first answer to this question:
https://stackoverflow.com/questions/2771676/django-datetime-issues-default-datetime-now

File: tracker/migrations/0002_streak.py (hunk 1 above)
Comment (ignaoya) [reply tracker/migrations/0002_streak.py]: Migration issue resolved.

**Pull request merged**